### PR TITLE
Manifest edit

### PR DESCRIPTION
**Pull request by Aeris1One**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
+        "fr": "Pterodactyl est un panel de gestion de serveurs de jeux (Minecraft, ARK, Rust,...)"
     },
     "version": "0.7.11",
     "url": "https://pterodactyl.io/",
     "license": "free",
     "maintainer": {
-        "name": "liberodark",
-        "email": "liberodark@gmail.com"
+        "name": "Pterodactyl Team",
+        "email": "support@pterodactyl.io"
```
Comment (Aeris1One) [inline manifest.json]: Not sure about the mail address to use, is it correct ?

Comment (yalh76) [reply manifest.json:14]: No it's the YunoHost Packager name and email to provide :)
So your name if you want to be the packager of this app

Comment (yalh76) [reply manifest.json:14]: But as you corrected it in #27, I will close that PR :)

**Comment by yalh76** [comment]
Done in #27

**Pull request closed without merging**